docs/solarsystem/pds/pds.rst

```diff
@@ -0,0 +1,116 @@
+.. _astroquery.solarsystem.pds:
+
+***********************************************************************************
+PDS Planetary Ring Node Queries (`astroquery.solarsystem.pds`)
+***********************************************************************************
+
+Overview
+========
+
+
+The :class:`~astroquery.solarsystem.pds.RingNodeClass` provides an
+interface to the ephemeris tools provided by the `NASA Planetary Data System's Ring Node System`_ hosted by SETI institute.
+
+Ephemeris
+-----------
+
+In order to query information for a specific Solar System body, a
+``RingNode`` object is instantiated and the :meth:`~astroquery.solarsystem.pds.RingNodeClass.ephemeris` method is called. The following example queries the
+ephemerides of the rings and small moons around Uranus as viewed from ALMA:
+
+.. code-block:: python
+
+   >>> from astroquery.solarsystem.pds import RingNode
+   >>> import astropy.units as u
+   >>> bodytable, ringtable = RingNode.ephemeris(planet='Uranus',
+   ...                 epoch='2024-05-08 22:39',
+   ...                 location = (-67.755 * u.deg, -23.029 * u.deg, 5000 * u.m))  # doctest: +REMOTE_DATA
+   >>> print(ringtable)
+         ring  pericenter ascending node
+                  deg          deg
+       ------- ---------- --------------
+           Six    293.129           52.0
+          Five    109.438           81.1
+          Four    242.882           66.9
+         Alpha    184.498          253.9
+          Beta     287.66          299.2
+           Eta        0.0            0.0
+         Gamma     50.224            0.0
+         Delta        0.0            0.0
+        Lambda        0.0            0.0
+       Epsilon    298.022            0.0
+
+``planet`` must be one of ['mars', 'jupiter', 'uranus', 'saturn', 'neptune', 'pluto'] (case-insensitive)
+
+.. code-block:: python
+
+   >>> bodytable, ringtable = RingNode.ephemeris(planet='Venus',
+   ...                 epoch='2024-05-08 22:39',
+   ...                 location = (-67.755 * u.deg, -23.029 * u.deg, 5000 * u.m)) # doctest: +IGNORE_EXCEPTION_DETAIL
```

Comment:
Is `+IGNORE_EXCEPTION_DETAIL` needed here? I would have thought the ellipsis in the traceback message should have been sufficient.